components/vf-content/vf-content.scss

```diff
@@ -135,6 +135,11 @@
     color: map-get($vf-colors-map, vf-color--grey);
     font-style: italic;
   }
+
+  .vf-video {
```

Comment:
Might be overengineering, but should we do `.vf-video + p` as this is to be used with vf-content where we wouldn't want extra spacing at the end of section ... or before and H2/H3, we wouldn't want space there, right?

Oh, and could you drop something in the changelog about this?

Reply:
ah, you've hit on something -- I guess we should cover H2/3/4/5/6 too an lists and alter accordingly.

I'll have a play and see what's a nice solution to resolve it.

Reply:
… because headings have top margin as well as bottom margin (if I recall correctly)

Reply:
👍